src/core/layout/qgslayoutobject.h

```diff
@@ -162,6 +162,7 @@ class CORE_EXPORT QgsLayoutObject: public QObject, public QgsExpressionContextGe
       MapYMax, //!< Map extent y maximum
       MapAtlasMargin, //!< Map atlas margin
       MapLayers, //!< Map layer set
+      MapCRS, //!< Map CRS
```

Comment:
```suggestion
      MapCrs, //!< Map CRS
```

You'll also need to move this to the end of the enum list, or it will re-assign the other existing entries (potentially breaking existing Python code)

Reply:
changed